test/integration/targets/callback_default/runme.sh

```diff
@@ -24,6 +26,11 @@ run_test() {
 	# Scrub deprication warning that shows up in Python 2.6 on CentOS 6
 	sed -i -e '/RandomPool_DeprecationWarning/d' "${OUTFILE}.${testname}.stderr"
 
+	# Printing task path includes absolute paths specific to the machine
+	# Replace any occurences of test folder path with TEST_FOLDER_PATH palceholder
+	sed -i -e "s#${TEST_FOLDER_PATH}#TEST_FOLDER_PATH#g" "${OUTFILE}.${testname}.stdout"
```

Comment:
Only do this when needed rather than for every single test.
```suggestion
	if [[ $testname == "display_failed_path" ]]; then
		# Printing task path includes absolute paths specific to the machine
		# Replace any occurences of $ANSIBLE_PLAYBOOK_DIR with TEST_FOLDER_PATH palceholder
		sed -i -e "s#${ANSIBLE_PLAYBOOK_DIR}#TEST_FOLDER_PATH#g" "${OUTFILE}.${testname}.stdout"
	fi
```

It may also be better to split this out into a separate function and call it directly rather than putting it inside this general purpose function.

Reply:
Done